docs/lit/examples/1_Examples.jl

```diff
@@ -52,6 +52,25 @@ plot(y, ylabel=P"mass in kilograms")
 
 plot(y, ylabel="")
 
+# ### Surround
+
+# If you prefer some other formatting over the round parentheses, you can
+# supply a keyword `surround`, which can be a number of different things:
+# `:round`,`:square`,`:curly` and `:slash` do what you might expect.
+# `:slashround` and the likes will combine them. `:verbose` will insert an " in
+# units of " between the label and the unit. `nothing` will remove the
+# parentheses.  For more customizatbility, a character, a string or a `Tuple`
+# of characters or strings can be supplied, which will be inserted between the
+# label and the unit, around the unit or before, between and after, depending
+# on the length of the tuple.
+
+plot(y,ylabel="mass",surround=:square)
```

Comment:
Could you add spaces after commas for consistency with the rest of the formatting?
```julia
plot(y, ylabel="mass", surround=:square)
```